storage/aptosdb/src/lib.rs

```diff
@@ -1207,21 +1207,51 @@ impl DbReader for AptosDB {
     }
 
     fn get_state_prune_window(&self) -> Result<Option<usize>> {
-        gauged_api("get_state_prune_window", || {
-            Ok(self
-                .pruner
-                .as_ref()
-                .map(|x| x.get_state_store_pruner_window() as usize))
-        })
+        if self.pruner.is_some() {
```

Comment:
use `if let Some(_) = self.pruner()` instead to avoid `unwrap()` later.

Reply:
Done